### ISSUE TITLE
A few bugs v2

### ISSUE THREAD

**Issue by Kimod** [Drutol/MALClient#210]
Since Guerra is fixing bugs, why not fix bugs that were submited more than one year ago? ^^

### Important issues (only UWP Version: `5.4.4.0` tho, on Android it works fine)
* Anime Recommendations/Suggestions don't work
* If you go to Reviews or Related, close them and open the off section, it will crash the app - https://i.imgur.com/MAvpgwW.gifv

### Bugs in MALClient UWP Version: `5.4.4.0`
* In history, there is "Edit" in every entry - https://i.imgur.com/V9vKh8T.png
* When I select in "Settings & More", "General", "Default anime sort values:" to "Last watched", it will sort by "Last watched", but after closing and navigating back to settings, "None" will be selected instead of "Last watched" - https://i.imgur.com/wBEocoQ.png
* If you compare profile picture of "Pita-kohai" in my friend list on Android and UWP, you can see, that it's better on Android, because it is vertically centered https://i.imgur.com/lSCa26Q.png
* Friend feeds doesn't support Rewatching - https://i.imgur.com/hNdrC4k.png
* In Friend feeds, if there are no hours or minutes, there is a double space between days and ago - https://i.imgur.com/eHL5qYC.png
* In Friend feeds, it looks like the determination of minute/minutes is dependent on the number of hours instead of minutes - https://i.imgur.com/w8VJ2ZD.png
* Some Films/OVAs in seasonal anime are in the wrong season - https://i.imgur.com/8skFxZQ.jpg
* If you add anime that you have already opened in right panel, trash can for deleting it won't appear, only after closing and oppening the right panel again - https://i.imgur.com/L5MOk5v.gifv
* If you delete anime from your list while you have "Seasonal anime" opened, the anime will be deleted from "Seasonal anime" list - https://i.imgur.com/qOUCWN7.gifv
* Since you made the search bar in "Search" small, the title of the section remains from the page you were before - https://i.imgur.com/114aqF5.png
* Search by "Anime" has transparent background while search by "Characters" doesn't - https://i.imgur.com/LJnuKRr.png
* If you navigate from Character to anime, it doesn't create a back arrow - this isn't the issue. Now, if you navigate from character to another character - to create back arrow with character - and then click on anime, it now has back arrow and if you click on that arrow, it will skip one character - https://i.imgur.com/iUwozhN.gifv (in this gif Shinobu is skipped, which is unforgivable! >//<)
* All characters without VA have text "No voice actors have been added to this character. Help improve our database by searching for a voice actor, and adding this character to their roles." at the end of their description - https://i.imgur.com/BzzXoMH.png
* I think this is all you have to replace in synopsis (for example É in anime synopsis looks like &Eacute; in your app): `[['&Eacute;', 'É'], ['&times;', 'x'], ['&rsquo;', "'"],   ['&lsquo;', "'"], ['&hellip', '...'], ['&le', '<'], ['<;', '; '], ['&hearts;', '♥'], ['&mdash;', '-'], ['&eacute;', 'é'], ['&ndash;', '-'], ['&Aacute;', 'Á'], ['&acute;', 'à'], ['&ldquo;', '"'], ['&rdquo;', '"'], ['&Oslash;', 'Ø'], ['&frac12;', '½'], ['&infin;', '∞'], ['&agrave;', 'à'], ['&egrave;', 'è'], ['&dagger;', '†'], ['&sup2;', '²'], ['&#039;', "'"], ['Psi;', "Ψ"]]
` (I found it here: https://github.com/Nihilate/Roboragi/blob/master/roboragi/MAL.py#L125 )
* If you change the width of off section and then you close it by pressing X and then open it again, it will open with that width you had before closing, whereas if you close it by swiping, it will open with width based on your resolution - https://i.imgur.com/4fJt1GA.gifv
* 1 Word tags will after 37 letters cut off.. https://i.imgur.com/g8pQjTK.png In the picture, I have two tags "123456789101112131415161718192021222324252627282930" and "1234567891011121314151617181920212223 24252627282930" as you can see, the second tag continues on next line
* In images, when you click on "Save", the name of the picture won't be whole
  * If you use save, and the image's name is more than 30 characters long, it will only get 30 characters
* The first time you open the off section and click on "Watched episodes", the tab for selecting episodes won't be focused on the number you have watched (you can see that it will focus on second click and also you can see that in anime list it works flawlessly) https://i.imgur.com/EtDNvaF.gifv
* In profile, the comment section is the only thing there that doesn't have caption on the yolo panel on the left - https://i.imgur.com/YwWTDdN.png
* The first time you use forum search you can search for nothing - https://i.imgur.com/QBNanP1.gifv
* When you reload in Articles and quickly click on News, it will load News with Articles in it - https://i.imgur.com/kDlNxDU.png
* If you click on multiple messages in Messaging, it will load both conversations - https://i.imgur.com/MMzRLI7.png
* Clicking on back arrow, when you're in "My Recent Posts" doesn't do anything - https://i.imgur.com/mZqK8NC.png
* After clicking on "My Recent Posts", last thing that you were viewing will remain highlighted in hamburger - https://i.imgur.com/7O8c86B.png
* Related doesn't reload when you click on back arrow in the off section (you have to click on something in Related and then go back in it, it won't work if you click on bunch of stuff in your list) I can't provide gif, because Related keeps crashing.. :/
* Some people have 'more:' about them on website, your app doesn't have it - https://i.imgur.com/C3GB9u9.jpg , if there is problem with space, you could just add "more" button, like some characters have "spoilers"

### Bugs in MALClient Android Version: `1.3.8.0`
* When you un-pin post in forums while you have multiple pinned, it leaves blank space not only on the un-pined post - https://i.imgur.com/DZIBnea.png (I only unpinned one)
* When you update the episode count by "Watched episodes" dialog, the "+" and "-" won't respect, if it's the first or last episode ("-" should be grayed out, because you can't decrease number of episodes) - https://i.imgur.com/rtrzAbk.png
* In Seasonal anime, when you hold the floating button, flyout has white line at the bottom - https://i.imgur.com/VgxHmse.png (other floating button menus don't have it)
* Long tags are cut off - https://i.imgur.com/VMlQaa3.png
* When you aren't logged in and click on "Sign in" or "Settings & more", it won't highlight it in hamburger - https://i.imgur.com/mWxQ6Jl.png
* When you aren't logged in and click on something in hamburger you can't access (like search), it will still select it in hamburger - https://i.imgur.com/jTq3qSt.png
* "Soonest airing" in Seasonal anime is named "Air day" on Android - https://i.imgur.com/ATCvl9t.png
* When you click on "..." in hamburger, the second tap on back button won't close the hamburger, but instead close the app - https://i.imgur.com/EhlBsCP.png
* Keyboard issues
  * When you click on something in the "List source" dialog that isn't input or [Go!] button, the keyboard won't input anywhere - https://i.imgur.com/uqk64wV.png (the same happens in "Find user" in "Profile")
  * When you have keyboard extended (by clicking on search) and click on "Watched episodes" (ex. 7/24) (or Set score or Set status in ...), the keyboard won't input anywhere and will obscure space on the screen - https://i.imgur.com/jK5ixGh.png
  * Another keyboard remaining extended bug is when you are in Profile, click on "Comment..." input and then click on "Find user"... There has to be something done across the app, or should I just list all the places this happens..?
* Double click issues (clicking on two different things produces weird things)
  * like.. Clicking on Search and Calendar in hamburger produces Calendar with search bar - https://i.imgur.com/3uo1etS.png
  * or: Clicking on two different "..." in grid view will open both dialogs, but when you try to close them by pressing back button, it will only close one and the second click on back button will close the app
  * So... Isn't there something you can set appwide, to just dissable this "double clicking"?
* When you are in search by specific studio/genre subcategory and click on back button, it will take you to Anime list instead of the studio/genre selector
* When you extend floating button in Anime list and then open hamburger menu by draging it from side (not by clicking the hamburger icon), the floating button won't hide it's sub-bubbles - https://i.imgur.com/JROFpgN.png
* When you are loading an Article and swipe right (to News) and are fast enough, it will set the title of the article to "News" - https://i.imgur.com/wjmMLtb.png - maybe it would be better to dissable interacting with the app when something is loading, there is another issue when you are in anime list
* There is some problem with "Search" on android, when you're quickly switching from anime to manga to anime to manga..... like.. it makes duplicate entries - https://i.imgur.com/XAzVdR7.png
* When you extend keyboard in "Log in" page, it will move the image in background - https://i.imgur.com/NPvNiz1.png
* There isn't "Search away" or "We have come up empty..." in character search (like in anime/manga search)
* The grids in history page don't have anything on them (status/watched eps/score/tags/type), unlike in UWP version
* When you go to some anime by studio/genre subcategory, "Anime list" in hamburger becomes selected - https://i.imgur.com/a9boJgg.png
* My device's screen is too small to get to the "Save" anime cover button - https://i.imgur.com/ZInuczn.png (you can't even scroll to get to it)

### Bugs that Drutol has given up on fixing, but I will list them anyway ¯\\\_(ツ)_/¯ (UWP `5.4.4.0`)
* When you scroll down just a little bit past first review, it will extend upwards instead of downwards - https://i.imgur.com/Gq1yf0D.gifv
* If you extend one review in anime off section and then click on different anime and go to reviews, it will have one review extended (can't provide gif.. because reviews are crashing, but it's probably not fixed)
* The watermark above "Search away!" in "Anime Search" is above hamburger - https://i.imgur.com/zUTrQnI.gifv

### Live tiles (UWP `5.4.4.0`)
* If you pin anime from your list with your status and you change it (like you add more watched episodes, or change score), the live tile won't update
* If you pin anime where you select everything except Score and Big title, then the amount of watched episodes won't appear on the live tile, because it's on the same line as Score
* If you pin manga that you are "Reading", it will show as "Watching" in live tile
* When you are pinning manga, "Watched" should be "Read" and there shouldn't be "Air day"

**Comment by Guerra24** [comment]
*"Search away!" in "Anime Search" is above hamburger* - This is why you should never ever use transform for object position, they're evil 👿. Let the layout manager handle it, so Drutol don't use them please.

**Comment by Drutol** [comment]
Did I? Whoopsies daisies then... it's been sooo long. xd

**Comment by Kimod** [comment]
Just saw your comment regarding MALClient on r/anime @Drutol and thanks to it, your thread on r/Dakimakuras. Teehee :p
I'm really liking the Jibril one ^^, but other girls from AoKana look really lovely too. Shame that the VN is only translated to the first chapter, so that's stopping me from discovering them >.>
I figured I could share my daki, since I have one too 😇. It started with a figurine and then I just couldn't resist. So, here she is - https://i.imgur.com/PahwIIL.jpg
I'm still kinda sad, that the society kinda looks down upon us, but what can we do.. I guess it's just better to not embrace it at all.

**Comment by Drutol** [comment]
We should rather switch to discord with this topic! Come join us ^^

**Comment by Kimod** [comment]
Comment that lists all fixed bugs:

### Fixed in Android Version: `1.4.13.0`
 * Start and End dates are now in MM.DD.YYYY format, while My start is in YYYY.MM.DD format. So the bug here is that there are two different formats. If the MM.DD.YYYY is intentional, could you make a setting for DD.MM.YYYY? Or just make everything YYYY.MM.DD?..
    *   ✅ I've tried to rework dates but spaghetti is too stronk... yyyy/mm/dd it is.
* If you try to refresh by pulling in "New message" it will crash the app
  * ✅ Fixed
* I think you should disable/hide "Make things black in black theme... Blah blah blah" when you have "Light theme" checked
  * ✅ I've added additional disclaimer, no point in hiding it I guess. Ideally I would have to place as a radio button in themes.

### Fixed in Android Version: `1.4.12.0`
* Default view modes for status in general settings display anime/manga list by the way Watching/Reading is set - https://i.imgur.com/r3bFWe9.png (Completed won't display as Detailed grid but as Grid)
* Anime "Bishoujo Senshi Sailor Moon SuperS" has more than 39 episodes listed in Episodes, but those were never released (or they are from next season)? Anyway, if you click on forum discusion for those episodes, it will crash the app, so just delist them? - https://i.imgur.com/BXCwg23.png
  * ✅ Fixed
* Notification with friend confirming my friend request is weird-ly formated. If I expand it, it's fine tho - https://i.imgur.com/9q3IpHv.png
  * 🤔 I guess I removed one redundant call setting notification properties. Let's see if that helps.
* When anime doesn't have all eps listed and you don't have the anime finished, in episodes, it will check that amount of episodes you have watched - https://i.imgur.com/BzgxbQ5.png (4 are listed, 4 are checked, 5 & 6 shouldn't be checked)
  * ✅ Fixed
* The amount of checked episodes is [amount of watched episodes] + 1 - https://i.imgur.com/f93Iv4w.png
  * ✅ Fixed
* When I'm in some person's details in landscape mode, when I scroll down, for a small part, the top part dissapears???? When I scroll up again, it will appear again and when I scroll down again, it will dissapear once again - https://i.imgur.com/VF1cKZk.png 
  * ✅ Hmm, I've changed the colour which was hard-coded to white to one matching the theme. It's the easiest way to make this screen make sense in landscape more space-wise.
* I guess this is intentional bcs it's unnecessarily complicated for such a small thing, but when anime has more than 100 eps, it won't list them, because on website, they are on next page - https://i.imgur.com/jYMfnkj.png
  * ✅ Fixed
* When you search for something and open an anime, it won't have everything properly loaded (shorter synopsis, status, start and end date) you can refresh to properly load it tho - https://i.imgur.com/q8yLyHB.png
  * ✅ Fixed
* There isn't our profile picture in hamburger - https://i.imgur.com/zOrnwUB.png
  * ✅ Fixed
* In history, there is "Edit" in every entry - https://i.imgur.com/V9vKh8T.png
  * ✅ Fixed
* In Friend feeds, if there are no hours or minutes, there is a double space between days and ago - https://i.imgur.com/eHL5qYC.png
  * ✅ Fixed
* In Friend feeds, it looks like the determination of minute/minutes is dependent on the number of hours instead of minutes - https://i.imgur.com/w8VJ2ZD.png
  * ✅ Fixed
* All characters without VA have text "No voice actors have been added to this character. Help improve our database by searching for a voice actor, and adding this character to their roles." at the end of their description - https://i.imgur.com/BzzXoMH.png
  * ✅ Fixed

### Fixed in Android Version: `1.4.6.0`
* Clicking on two different "..." in grid view will open both dialogs, but when you try to close them by pressing back button, it will only close one and the second click on back button will close the app
  * ✔️ Fixed by >tweaking some popup menus

### Fixed in UWP Version: `5.5.0.0`
* If you go to Reviews or Related, close them and open the off section, it will crash the app - https://i.imgur.com/MAvpgwW.gifv
  * ✔️ Fixed
* If you compare profile picture of "Pita-kohai" in my friend list on Android and UWP, you can see, that it's better on Android, because it is vertically centered https://i.imgur.com/lSCa26Q.png
  * ✔️ Done
* Search by "Anime" has transparent background while search by "Characters" doesn't - https://i.imgur.com/LJnuKRr.png
  * ✔️ Fixed
  *  Okay, it doesn't change the name.
* The watermark above "Search away!" in "Anime Search" is above hamburger - https://i.imgur.com/zUTrQnI.gifv
  * ✔️ Fixed
